dotnet/src/SemanticKernel.Abstractions/SemanticFunctions/PromptTemplateConfig.cs

```diff
@@ -151,7 +83,7 @@ public class InputConfig
     [JsonPropertyName("completion")]
     [JsonPropertyOrder(4)]
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
-    public CompletionConfig Completion { get; set; } = new();
+    public AIRequestSettings? Completion { get; set; }
```

Comment:
Suggest changing the property to `RequestSettings` or `Settings` as well (Abstract away the Completion concept)

Reply:
That change is outside of the scope of this PR. The `PromptTemplateConfig` class needs a number of changes so they'll be made separately.